src/Enzo/enzo_EnzoMethodGrackle.hpp

```diff
@@ -198,6 +198,8 @@ class EnzoMethodGrackle : public Method {
 			    "local_calculate_temperature");
   }
 
+  void enforce_metallicity_floor(EnzoBlock * enzo_block) throw();
```

Comment:
Is this function actually used anywhere?

Reply:
I must have forgotten to copy over the line calling this function when preparing the branch for this PR. Added that back in. It's now called in `compute_` just after `local_solve_chemistry`